CMakeLists.txt

```diff
@@ -54,7 +54,10 @@ message(STATUS "CMAKE_INSTALL_RPATH: ${CMAKE_INSTALL_RPATH}")
 # based on which build options are enabled
 set(PLUGIN_SRC_FILES SvalinnPlugin.cpp)
 set(PLUGIN_LINK_LIBS ${CUBIT_LIBS})
-
+SET(HDF5_USE_STATIC_LIBRARIES ON)
```

Comment:
It looks like the Linux build relies on shared libs.  What does this do?

Reply:
this is an artifact... that should not be there cleaning